tree/ntuple/v7/test/CMakeLists.txt

```diff
@@ -33,10 +33,10 @@ ROOT_ADD_GTEST(ntuple_types ntuple_types.cxx LIBRARIES ROOTDataFrame ROOTNTuple
 ROOT_ADD_GTEST(ntuple_view ntuple_view.cxx LIBRARIES ROOTDataFrame ROOTNTuple MathCore CustomStruct)
 ROOT_ADD_GTEST(ntuple_zip ntuple_zip.cxx LIBRARIES ROOTDataFrame ROOTNTuple MathCore CustomStruct)
 
-ROOT_ADD_GTEST(rfield_class rfield_class.cxx LIBRARIES ROOTDataFrame ROOTNTuple MathCore CustomStruct)
+ROOT_ADD_GTEST(rclass_field rclass_field.cxx LIBRARIES ROOTDataFrame ROOTNTuple MathCore CustomStruct)
```

Comment:
I think here I would either leave the `rfield` prefix or change it everywhere to type first, including string.  From my current gut feeling, I'd have a slight preference to leave the `rfield` prefix to keep all the field tests alphabetically together.

Reply:
You got it, I'll change them back to `rfield`. I kept `rfield_string` because it reminded me a bit of `RField<std::string>`.